xrootd.spec

```diff
@@ -36,7 +41,8 @@ cmake ../ \
   -DENABLE_KRB5=TRUE \
   -DENABLE_READLINE=FALSE \
   -DENABLE_CRYPTO=TRUE \
-  -DCMAKE_SKIP_RPATH=TRUE
+  -DCMAKE_SKIP_RPATH=TRUE \
+  -DENABLE_PYTHON=TRUE -DPYTHON_LIBRARY=$PYTHON_ROOT/lib/libpython2.7.so
```

Comment:
Styling. `-DENABLE_PYTHON=TRUE` should be on it's own line.
Btw, why we enable Python support here?
Instead of `-DPYTHON_LIBRARY` you can use `CMAKE_PREFIX_PATH`. See: https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_9_1_X/gcc700/root.spec#L139